<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>599</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Expt-o Morris</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="chkCor">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>274</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cor</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkInfravermelho">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>274</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Infravermelho</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk3d">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>274</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblIR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl3D">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCapturar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Capturar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFPSCor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FPS: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFPSIR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FPS: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFPS3D">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FPS: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrir">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFPSIR_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>107</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Identificação</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtNome">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <zorder>chkCor</zorder>
   <zorder>chkInfravermelho</zorder>
   <zorder>chk3d</zorder>
   <zorder>lblCor</zorder>
   <zorder>lblIR</zorder>
   <zorder>lbl3D</zorder>
   <zorder>btnCapturar</zorder>
   <zorder>lblFPSCor</zorder>
   <zorder>lblFPSIR</zorder>
   <zorder>lblFPS3D</zorder>
   <zorder>lblPlay</zorder>
   <zorder>btnAbrir</zorder>
   <zorder>lblFPSIR_2</zorder>
   <zorder>textEdit</zorder>
   <zorder>txtNome</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
